src/BrowserDetector.js

```diff
@@ -1,9 +1,6 @@
-import memoize from 'lodash/memoize';
 
-export const isFirefox = memoize(() =>
-  /firefox/i.test(navigator.userAgent)
-);
+const _isSafari = Boolean(window.safari);
+const _isFirefox = !_isSafari && /firefox/i.test(navigator.userAgent);
```

Comment:
No. It was stupid. For some reason I had the impression that this file was ment to be as optimized as possible, and I assumed that for all iOS-devices out there it would be a tiny bit faster to not have to execute that regex.

It could also be an idea to not use an regexp for this simple task at all:

```javascript
const _isFirefox = navigator.userAgent.toLowerCase().indexOf('firefox') !== -1;
```